pkg/cmd/secret/set/set.go

```diff
@@ -175,7 +185,11 @@ func setRun(opts *SetOptions) error {
 	if orgName != "" {
 		err = putOrgSecret(client, host, pk, *opts, encoded)
 	} else {
-		err = putRepoSecret(client, pk, baseRepo, opts.SecretName, encoded)
+		if envName != "" {
```

Comment:
Not sure if it's the most idiomatic way of writing this, let me know if you prefer it as:

```go
	if orgName != "" {
		err = putOrgSecret(client, host, pk, *opts, encoded)
	} else if envName != "" {
		err = putEnvSecret(client, pk, baseRepo, envName, opts.SecretName, encoded)
	} else {
		err = putRepoSecret(client, pk, baseRepo, opts.SecretName, encoded)
	}
```

Reply:
Collapsing that nested conditional nto the else as you suggested would be a welcome change; it's so minor I'll just push a commit before I merge.